src/main.rs

```diff
@@ -198,10 +218,14 @@ fn configure_app(app: App<MiniserveConfig>) -> App<MiniserveConfig> {
         }
     };
 
+    let route_url = "/".to_string();
```

Comment:
Maybe call this `base_route` to communicate the intention?

Reply:
And in fact, you can probably just remove this as you don't really need it. Just put it directly in your format string at `let full_route`.